components/Header/header.js

```diff
@@ -0,0 +1,22 @@
+function modelFunction(){
+    var x=document.getElementById('modal');
+    var y=document.getElementById('symbol');
+    console.log(x)
+    if(x.className==='list-parent-right'){
+     x.className+="modalMobile";
+     y.className+="remove";
+     console.log(x,y)
```

Comment:
again i think console.log()s are used for debugging , you can remove them from code

Reply:
yes it is of no use here, you can remove this